source/ranges.tex

```diff
@@ -5009,8 +5009,8 @@
 \pnum
 Let \tcode{\placeholder{inner-range}} be:
```

Comment:
```suggestion
Let \exposid{inner-range} be:
```

Reply:
This is really a placeholder for some code, not an exposition-only member. Leaving as-is.